frontend/src/test/scala/bloop/CliSpec.scala

```diff
@@ -154,7 +154,7 @@ object CliSpec extends BaseSuite {
   }
 
   test("fail at non-sensical host address") {
-    checkInvalidAddress("localhos")
+    checkInvalidAddress("localhost-")
```

Comment:
This is probably one of the more "interesting" changes. (Very few of the changes here are remotely interesting.)

Reply:
LGTM, checking for `localhost-` should be the same error as checking for `localhos`.